adserver/analyzer/models.py

```diff
@@ -68,5 +62,31 @@ def save(self, *args, **kwargs):
         self.full_clean()
         return super().save(*args, **kwargs)
 
+    class Meta:
+        abstract = True
+
+
+class AnalyzedUrl(BaseAnalyzedUrl):
+    """Analyzed keywords for a given URL."""
+
+    publisher = models.ForeignKey(
+        Publisher,
+        help_text=_("Publisher where this URL appears"),
+        on_delete=models.CASCADE,
+    )
+
     class Meta:
         unique_together = ("url", "publisher")
+
+
+class AnalyzedAd(BaseAnalyzedUrl):
+    """Analyzed keywords for a given URL."""
+
+    advertiser = models.ForeignKey(
```

Comment:
So the idea here when we want to check if a particular flight/ad landing page URL is in the DB is to just normalize the URL (strip UTMs) and then do a DB lookup with the URL and the advertiser? That seems reasonable.